README.md

```diff
@@ -136,7 +139,7 @@ Following the corresponding link [Installation Guide document](https://github.co
 	1. INCLUDECADMIUM=-I lib/cadmium/include
     2. Update the relative path to cadmium/include from the folder where the makefile is. You need to take relative path into account where you have copied the folder during the installation process
 
-		> Example: INCLUDECADMIUM=-I lib/cadmium/include
+		> Example: git clone --recurse-submodules -b submodule_simulator_modification https://github.com/abhinavachanta/AlternateBitProtocolSimulator.git AlternateBitProtocol
```

Comment:
i followed the steps mentioned in README and also used this link to initiate submodules, but it did not work. You need to include the right way.

Reply:
To fix the problem i suggest that you use the following command (git submodule update --init
) to invoke and update the submodules instead of the command you have in line 131. Please addresses this issue and other points to be able to review your code.